spirv_msl.cpp

```diff
@@ -11192,7 +12724,9 @@ void CompilerMSL::analyze_argument_buffers()
 				auto &var = set<SPIRVariable>(var_id, uint_ptr_type_id, StorageClassUniformConstant);
 				set_name(var_id, "spvBufferSizeConstants");
 				set_decoration(var_id, DecorationDescriptorSet, desc_set);
-				set_decoration(var_id, DecorationBinding, kBufferSizeBufferBinding);
+				/* UE Begin Change: Move this to the front of IABs for convenience of the runtime */
```

Comment:
hm?

Reply:
The add_msl_resource_binding API lets you rebind these things to an appropriate `[[id()]]` you'd like, even for these internally created resources. Please remove this hardcoding.

Reply:
This still looks like a hack.